src/DynamoCore/DynamoCore.csproj

```diff
@@ -70,10 +70,6 @@
       <Project>{c0d6dee5-5532-4345-9c66-4c00d7fdb8be}</Project>
       <Name>DesignScriptBuiltin</Name>
     </ProjectReference>
-    <ProjectReference Include="..\Libraries\DynamoUnits\Units.csproj">
-      <Project>{6e0a079e-85f1-45a1-ad5b-9855e4344809}</Project>
-      <Name>Units</Name>
-    </ProjectReference>
```

Comment:
Was this an extraneous dependency that wasn't being used by DynamoCore?

Reply:
yep - there is a new feature (new to me?) in vs 2022 - remove unused references.